app/store/configureStore.js

```diff
@@ -1,6 +1,12 @@
 // @flow
-if (process.env.NODE_ENV === 'production') {
-  module.exports = require('./configureStore.prod'); // eslint-disable-line global-require
-} else {
-  module.exports = require('./configureStore.dev'); // eslint-disable-line global-require
-}
+import configureStoreDev from './configureStore.dev';
```

Comment:
Will `webpack` remove this when "tree-shaking" for production?

This does add an unnecessary step of importing both modules first then deciding what to do with them.

Reply:
Yea, webpack will remove it. We are using `EnvironmentPlugin`